src/System.Private.ServiceModel/tests/Scenarios/SelfHostWcfService/WcfService.cs

```diff
@@ -234,5 +236,46 @@ public XmlCompositeType GetDataUsingXmlSerializer(XmlCompositeType composite)
             composite.StringValue = composite.StringValue;
             return composite;
         }
+
+        public Stream GetStream(string data)
+        {
+            return StringToStream(data);
+        }
+
+        public bool UploadStream(Stream stream)
+        {
+            string data = StreamToString(stream);
+
+            if (!string.IsNullOrWhiteSpace(data))
+            {
+                return true;
```

Comment:
... and on that note, maybe you need to define some magic stream contents, or return the stream back with the original data to test the StreamedResponse.